wagtail/admin/templates/wagtailadmin/pages/index.html

```diff
@@ -2,7 +2,7 @@
 {% load wagtailadmin_tags i18n %}
 
 {% block header %}
-    {% include 'wagtailadmin/shared/headers/slim_header.html' with title=header_title search_url=index_results_url %}
+    {% include 'wagtailadmin/shared/headers/page_viewset_header.html' with title=header_title search_url=index_results_url %}
```

Comment:
Let's try to keep using `slim_header.html` if possible 
```suggestion
    {% include 'wagtailadmin/shared/headers/slim_header.html' with title=header_title search_url=index_results_url %}
```